extensions/VisualEditor/wikia/modules/ve/ui/styles/images/icons/image.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I believe we use this one https://petercollingridge.appspot.com/svg-editor but I'm not sure about exact options - you would have to play with it a bit and look what output it generates.